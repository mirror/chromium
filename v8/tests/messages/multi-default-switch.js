switch (foo) {
  default: break;
  default: break;
}
