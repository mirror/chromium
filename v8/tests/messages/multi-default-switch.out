multi-default-switch.js:3: Uncaught SyntaxError: More than one default clause in switch statement
  default: break;
  ^^^^^^^
